backend/runners/fetch_tiger_data.py

```diff
@@ -0,0 +1,114 @@
+"""
```

Comment:
Why is this here?

Reply:
The output of this script is used by the `generate_representative_points.py` script. 

Happy to exclude both if you don't think they belong here, but I'm not sure what the value is of having one without the other.

Reply:
I was just curious as why it did not already exist

Reply:
Stolen shamelessly from RPG.

Reply:
👌